jdaviz/configs/default/plugins/subset_plugin/subset_plugin.py

```diff
@@ -85,6 +90,27 @@ def __init__(self, *args, **kwargs):
         self.subset_states = []
         self.spectral_display_unit = None
 
+        if self.multiselect:
+            self.display_sky_coordinates = False
+        else:
+            self.display_sky_coordinates = (self.app._link_type == 'wcs' if hasattr(self.app, '_link_type') else False)
+
+    def _on_link_update(self, *args):
+        """When linking is changed pixels<>wcs, change display units of the
+           subset plugin from pixel (for pixel linking) to sky (for wcs linking).
+           If there is an active selection in the subset plugin, push this change
+           to the UI upon link change by calling _get_subset_definition, which
+           will re-determine how to display subset information."""
+
+        if self.multiselect:
+            self.display_sky_coordinates = False
+            return
```

Comment:
the entire section is hidden when in multiselect, right?  Should we just leave the traitlet as it was?  If this does need to become False and/or return, then we also need to observe multiselect to restore its value (and re-call `_get_subset_definition`) when multiselect is disabled.

Reply:
good point, removed the check for multiselect.